src/pages/create-new-checkin-page/steps/review-previous-checkin-step.jsx

```diff
@@ -23,6 +23,24 @@ const propTypes = {
 };
 
 const ReviewPreviousCheckinStep = ({ checkin, onComplete }) => {
+  // TODO Figure out how to make `checkin` exist on the backend for SSR
+  // NOTE Temporary guard for lack of state in SSR
```

Comment:
My advice for this is to render a loading indicator on both the server and the client, and then trigger a load when the component mounts on the client side. You can use `useEffect` to trigger a load in your sagas.

The advantage is that you'll be able to serve cached, static content from the server (which is very fast), and you can still take advantage of client-side localStorage caching.